.github/workflows/publish.yml

```diff
@@ -24,5 +24,5 @@ jobs:
       - run: ./Build.ps1
         shell: pwsh
         env:
-          NUGET_SOURCE: ${{ secrets.NUGET_SOURCE }}
-          NUGET_API_KEY: ${{ secrets.WORKLEAP_NUGET_API_KEY }}
\ No newline at end of file
+          NUGET_SOURCE: ${{ secrets.NUGET_GSOFTDEV_FEED_URL }}
```

Comment:
Given publish is triggered when things are merged into main or a tag is created, do we really want to push to the GSoft dev feed? I'd think either the GSoft or Workleap feed no?

Reply:
Currently, we haven't tagged a `0.1.0` version and that it is currently work in progress, we thought publishing to `gsoftdev` would make sense at the moment. We'll eventually change this to publish on the nuget org feed.